packages/types/api.d.ts

```diff
@@ -841,6 +841,12 @@ declare abstract class BaseParser {
    */
   protected canTokenTypeBeInsertedInRecovery(tokType: TokenType): boolean
 
+  /**
+   * By default all token types may be deleted. This behavior may be overridden in inheriting parsers.
+   * The method receives the expected token type.
+   */
+  protected canTokenTypeBeDeletedInRecovery(tokType: TokenType): boolean
```

Comment:
useful to link to the docs here?
- https://chevrotain.io/docs/tutorial/step4_fault_tolerance.html#single-token-deletion

Reply:
Makes sense. I will add a note to the docs.